    @Test
    public void testConverting() throws Exception {
        builder.set("integer", "1");
        builder.buildFeature("fid");
        try {
            builder.set("integer", "foo");
            Assert.fail("should have failed");
        } catch (Exception e) {
        }
    }

